upgrade-notes/_layout.md

```diff
@@ -0,0 +1,50 @@
+# UPGRADING FROM 14.x to 15.0
```

Comment:
I think the current UPGRADE-15.0.md should be deleted

Reply:
Deleted.